cmd/lotus/daemon.go

```diff
@@ -27,10 +27,6 @@ var DaemonCmd = &cli.Command{
 	Name:  "daemon",
 	Usage: "Start a lotus daemon process",
 	Flags: []cli.Flag{
-		&cli.StringFlag{
```

Comment:
Removing this flag breaks pond: https://github.com/filecoin-project/lotus/blob/master/lotuspond/spawn.go#L49